AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1174,6 +1174,7 @@ lightnovelpub.com,lightnovelworld.com#$#.chapter-content { -webkit-touch-callout
 mediafire.com#$#.nonDLContentWrap { min-height: 1300px !important; }
 lanacion.com.ar#$#.sidebar__aside { background: none!important; }
 goha.ru#$##site { margin-top: 150px !important; }
+sematext.com#$#.navbar-fixed-top { top: 0 !important; }
```

Comment:
why not to popups.txt?

Reply:
I've only moved navbar so the gap won't show

Reply:
@Alex-302

Reply:
it is related to `#wpfront-notification-bar-spacer`